Services/Services/CandidateService.cs

```diff
@@ -173,19 +183,21 @@ public int AddHRNote(int candidateId, CandidateAddHRNoteDTO dto)
                 catch (Exception ex)
                 {
                     _logger.LogError("Error when adding HR note to candidate: " + ex);
-                    return -2;
+                    ErrorMessage = ErrorMessageHelper.ErrorAddingHRNoteToCandidate;
+                    return -1;
                 }
-
+                ErrorMessage = "";
                 return 1;
```

Comment:
Return ID of the note

Reply:
We don't have a dedicated database table for notes. Currently the HR Note and Tech Note (with their respective scores) are held in the Candidate table. Keeping multiple notes for one candidate would require some massive changes to what we have right now.